sdk/lib/opentelemetry/sdk/trace/export/noop_span_exporter.rb

```diff
@@ -13,20 +13,33 @@ module Export
         # recorded data for sampled spans in their own format.
         #
         # To export data an exporter MUST be registered to the {Tracer} using
-        # a {SimpleSampledSpanProcessor} or a {BatchSpanProcessor}.
+        # a {SimpleSpanProcessor} or a {BatchSpanProcessor}.
         class NoopSpanExporter
+          def initialize
+            @stopped = false
+            @mutex = Mutex.new
```

Comment:
We don't need this `Mutex`. The Ruby memory model essentially provides atomic updates of instance variables. Had we been updating or reading multiple dependent variables, a `Mutex` would have been necessary, but for this simple boolean, it isn't.

Reply:
Thank you for the missing context!
This is done now.